src/FFNNWaveFunction.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef FFNN_WAVE_FUNCTION
+#define FFNN_WAVE_FUNCTION
+
+#include "WaveFunction.hpp"
+#include "FeedForwardNeuralNetwork.hpp"
+
+#include <stdexcept>
+
+
+
+class FFNNWaveFunction: public WaveFunction{
+
+private:
+    FeedForwardNeuralNetwork * _ffnn;
+
+public:
+    // --- Constructor and destructor
+    // IMPORTANT: The provided ffnn should be ready to use (connected) and have the first, second and variational derivatives substrates
```

Comment:
I would add another constructor, where you only pass number of layer and units and the ffnn gets built by the constructor. But that's just a feature request.

Reply:
So if you want a default style nn quickly, you use the second constructor. For custom built NN, pass it.

Reply:
mmm... no, I wouldn't do it here. If that's the case, then we should add it to the FFNN library.
You can add this point to the "Make FFNN more user friendly" Trello's card